users/README.md

```diff
@@ -0,0 +1,26 @@
+# Users
+
+```
+{
+  'id': string,
+  'username': string,
+  'first_name': string,
+  'last_name': string,
+  'email': string,
+  'phone': number,
+  'yoe': number,
+  'company': string,
+  'designation': string,
+  'img': string,
+  'github_id': string,
+  'linkedin_id': string,
+  'twitter_id': string,
+  'instagram_id': string,
+  'website': string,
+  'github_display_name': string,
+  'isMember': boolean,
+  'userType': string,
+  'tokens': {},
+  'badges': []
```

Comment:
Please add `status` as string here, which will contain the current status of user (ooo, active, idle)

Reply:
Added status: string